index.html

```diff
@@ -307,6 +307,57 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="about flex-column">
```

Comment:
Good job so far. Please try to match the Figma design as much as possible🙏. Your design is currently missing a lot of the styles applied to the form. Ensure to use the same font family, weight and color. And apply ample padding to each of the elements so it looks aesthetically pleasing 😎

Your design:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/37116322/218154946-c04a4ae8-d550-46d0-9f1b-073a346b2b25.png">

Figma design:
<img width="131" alt="image" src="https://user-images.githubusercontent.com/37116322/218155030-c7c526c9-0c4e-4e02-ab42-9d031c8d6a99.png">